extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -1,5 +1,8 @@
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index';
-@import 'skins/shared/color.scss';
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+@import 'skins/oasis/css/core/breakpoints-variables';
 
 .WikiaPage .page-header {
```

Comment:
@weronika added it in a commit named 
`fixed selector for PageHeader for css styling to take effect` (https://github.com/Wikia/app/commit/a391b4d0db4eeab79528185365d18eeb31feed47) so I suppose we need it :)